### PR TITLE
[v6r18][Techno Preview] New logging system

### PR DESCRIPTION
**Pull request by aldbr**
This system is independent of the rest, and does not modify any existing behaviour. 
It was agreed to merge it into v6r18 during the BILD meeting so that LHCb can start certifying it.

### CODE REVIEW

**Comment by chaen** [comment]
Just to say that I already thoroughly reviewed it. Some more changes will come, but we need this in to start seeing it used in certification.

File: FrameworkSystem/private/standardLogging/Backend/AbstractBackend.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+  In this way, we have an object composed by one handler and one formatter name. 
+  The purpose of the object is to get cfg options to give them to the handler,
+  and to set the format of the handler when the display must be changed. 
+  """
+
+  def __init__(self, handler, formatter):
+    """
+    Initialization of the backend.
+    :params _handler: handler object from 'logging'. Ex: StreamHandler(), FileHandler()...
```
Comment (andresailer) [inline FrameworkSystem/private/standardLogging/Backend/AbstractBackend.py]: There needs to be  a newline before the `:params ...` block

**Comment by fstagni** [comment]
First of all: congrats @aldbr , impressive work! 

My few questions/comments:
- do the tests work with BOTH gLogger and "gLogging"?
- The unit test files should be called "Test_abc.py" in order to be run by Travis/Jenkins (while all those here are called "Testabc.py"), is the reason to be found in the answer of my question above?
- can we have a CS flag for enabling gLogging?

**Comment by aldbr** [comment]
Thank you ! 

+ No, the tests are made to work with "gLogging". The tests will fail with gLogger. 
+ In this way, I think it is not a good idea to let Travis/Jenkins runs these tests.
+ Yes, I think it is possible, where do I have to put this flag exactly ?

**Comment by chaen** [comment]
This PR seems good enough to me so that it is merged. Anyone else wants to rubber stamp it ? @atsareg can you please include it in the next patch release so we can do the testing in certification

**Pull request merged**